contxt/cli/main.py

```diff
@@ -26,11 +26,12 @@ def get_command(self, ctx, name):
 
 
 @click.group(cls=Cli, context_settings=dict(help_option_names=["-h", "--help"]))
+@click.option("--env", required=False, default="production")
 @click.version_option(__version__, "-v", "--version")
 @click.pass_context
-def cli(ctx: click.Context) -> None:
+def cli(ctx: click.Context, env: str) -> None:
     """Contxt CLI"""
-    ctx.obj = Clients(auth=CliAuth())
+    ctx.obj = Clients(auth=CliAuth(env=env), env=env)
```

Comment:
```suggestion
    ctx.obj = Clients(env=env)
```
and then we can add the following to the `Clients` class:

```python
@cachedproperty
def auth(self) -> CliAuth:
    return CliAuth(env=self.env)
```